sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -111,48 +110,6 @@ def test_pardo(self):
              | beam.Map(lambda e: e + 'x'))
       assert_that(res, equal_to(['aax', 'bcbcx']))
 
-  def test_pardo_metrics(self):
```

Comment:
Was this redundant or something? I don't fully understand this deletion.

Reply:
I am guessing you just want test_metrics to be the single metric test inhereted by all the other test classes.